modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/transition/bulk/select-transition-step/SelectTransitionStepCard.es.js

```diff
@@ -0,0 +1,239 @@
+				...transitionTasks.filter(
+					({workflowTaskId}) =>
+						!tasks.some(({id}) => id === workflowTaskId)
+				),
+				...tasks.map(({id}) => ({
+					comment,
+					transitionName: selectedTransition,
+					workflowTaskId: id,
+				})),
+			],
+		});
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [comment, selectedTransition, tasks]);
+
+	return (
+		<>
+			<ClayPanel.Body className="transition-panel-body">
+				<div className="table-container">
+					<div
+						className={`table-visualization ${
+							showAll ? 'table-visualization-show-all' : ''
+						}`}
+					>
+						<ClayList className={showAll ? 'table-show-all' : ''}>
+							<ClayList.Item
+								flex
+								style={{backgroundColor: '#F7F8F9'}}
+							>
+								<ClayList.ItemField style={{width: '20%'}}>
+									<ClayList.ItemTitle>
+										{Liferay.Language.get('id')}
+									</ClayList.ItemTitle>
+								</ClayList.ItemField>
+
+								<ClayList.ItemField expand>
+									<ClayList.ItemTitle>
+										{Liferay.Language.get('item-subject')}
+									</ClayList.ItemTitle>
+								</ClayList.ItemField>
+							</ClayList.Item>
+
+							{tasks.map(
+								({
+									id,
+									objectReviewed: {assetTitle, assetType},
+									workflowInstanceId,
+								}) => (
+									<ClayList.Item
+										className="list-item-no-border"
+										flex
+										key={id}
+									>
+										<ClayList.ItemField
+											style={{width: '20%'}}
+										>
+											<ClayList.ItemTitle>
+												{workflowInstanceId}
+											</ClayList.ItemTitle>
+										</ClayList.ItemField>
+
+										<ClayList.ItemField expand>
+											<ClayList.ItemText>
+												{`${assetType}: ${assetTitle}`}
+											</ClayList.ItemText>
+										</ClayList.ItemField>
+									</ClayList.Item>
+								)
+							)}
+						</ClayList>
+					</div>
+
+					{tasks.length > 10 && (
+						<span
+							className="mt-4 show-all-button"
+							data-testid="showAllButton"
+							onClick={() => {
+								setShowAll(!showAll);
+							}}
+						>
+							{!showAll
+								? Liferay.Language.get('show-all')
+								: Liferay.Language.get('show-less')}
+						</span>
+					)}
+				</div>
+
+				<div
+					className={`${
+						errors[cardIndex] && onGoing ? 'has-error' : ''
+					} ml-3`}
+				>
+					<label
+						className="control-label"
+						htmlFor={`transitionSelect${cardIndex}`}
+					>
+						{`${Liferay.Language.get('transition-to')} `}
+
+						<ClayIcon
+							className="reference-mark text-warning"
+							symbol="asterisk"
+						/>
+					</label>
+
+					<ClaySelect
+						aria-label="Select Label"
+						className="error-field"
+						data-testid="transitionSelect"
+						id={`transitionSelect${cardIndex}`}
+						onChange={handleChange}
+					>
+						<ClaySelect.Option label="" value="" />
+
+						{nextTransitions &&
+							nextTransitions.length > 0 &&
+							nextTransitions.map(nextTransition => (
```

Comment:
What do you think about this:
nextTransitions has a defaultValue as Array, you could to use only nextTransitions.map(...).